scss/components/_search-bar.scss

```diff
@@ -0,0 +1,132 @@
+// Search Bar
+//
+// .combo--search--large      - 6rem tall
+//
+// Markup:
+// <div class="slab bg--primary">
+//   <div class="container">
+//   <div class="combo combo--search {{modifier_class}}">
+//     <input type="text" class="combo__input">
+//     <button type="submit" class="button--primary combo__button">
+//      <span class="u-visually-hidden">Search</span>
+//     </button>
+//   </div>
+// </div>
+// </div>
+//
+// Styleguide components.search-bar
+
+.combo--search {
+  .combo__button {
+    background-image: url('../img/i-search--primary.png');
+    background-image: url('../img/i-search--primary.png');
+    background-repeat: no-repeat;
+    background-position: center center;
+  }
+
+  @include media($med) {
+    .combo__input {
+      width: calc(100% - 4.6rem);
+    }
+
+    .combo__button {
+      width: 4rem; 
+    }
+  }
+}
+
+.combo--search--large {
+  @include media($med) {
+    height: 6rem;
+    .combo__input {
+      font-size: 2.8rem;
+      height: 6rem;
+      width: calc(100% - 6.6rem);
+    }
+
+    .combo__button {
+      height: 6rem;
+      width: 6rem;
+    }
+  }
+}
+
+// Typeahead styles
```

Comment:
I've been trying to think of ways to deal with classes we can't control. One would be to have a `vendor-specific/typeahead file`. Just wanted to bring this up though, I don't think changes should be made now.